File: TRBusinessLayer/DataObjects/TaxReportWrapper.cs
using System;
using System.Collections.Generic;


namespace TRBusinessLayer.DataObjects
{
    public class TaxReportWrapper : AbstractDto
    {      
        public List<DropDownItem> AvailTaxTypes { get; set; }
        public List<DropDownItem> AvailVoucherId { get; set; }
        public List<DropDownItem> AvailStartPeriod { get; set; }
        public List<DropDownItem> AvailEndPeriod { get; set; }
    }
}

File: TRClient/src/app/Entities/TaxReportWrapper.ts
import {DropDownItem} from './dropDownItem';
import {Awrapper}     from './Awrapper';

export class TaxReportWrapper extends Awrapper  {
 
    AvailTaxTypes: DropDownItem[];

    AvailVoucherId: DropDownItem[];
    AvailStartPeriod: DropDownItem[];
    AvailEndPeriod: DropDownItem[];
}